src/components/AppHeader.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <header>
+    <div class="header-wrapper">
+      <div class="container">
+        <nav>
+          <div class="nav-wrapper">
+            <a
+              class="logo-link"
+              href="/"
+            >
+              <img
+                class="logo-image"
+                src="../assets/logo.svg"
+                alt="logo"
+                width="50"
+              >
+              <span class="logo-title">CINEMA-PROJECT</span>
+            </a>
+          </div>
+        </nav>
+      </div>
+    </div>
+    <div class="search-wrapper">
+      <div class="container">
```

Comment:
you have 2 wrappers for a component i think 1 is enough
You have  `.search-wrapper{padding: 10px 0;}` and `.container {padding: 0 20px;}`

Just add `padding: 10px 20px;` to one of them and delete another one.

But i think all styles for Search component, must be in SearchForm.vue
Same thing for `.header-wrapper` and `.container `

Reply:
.container is usually used to limit the width of the content. .container will be in allmost in all main blocks such as <main> <footer> <header> <section>. Wrappers i need to set background-color fol full width of screen. I try to avoid styling semantic tags. But i removed .container {padding: 0 20px;}, because Bootstrap gave me this option. I moved SearchForm wrappers in to it.

Reply:
You can set `.container` for `nav` tag and remove one parent `div`